src/common/util.cc

```diff
@@ -274,7 +274,7 @@ string cleanbin(bufferlist &bl, bool &base64)
 {
   bufferlist::iterator it;
   for (it = bl.begin(); it != bl.end(); ++it) {
-    if (iscntrl(*it))
+    if (iscntrl(*it) && !isspace(*it))
```

Comment:
No, you were right I use it in dump_shard() to determine whether to use Base64 encoding or not.